paper/paper.md

```diff
@@ -0,0 +1,86 @@
+``CapacityExpansion`` provides simple integration of (clustered) time-series, geographical, cost, and technology input data. The software features a modular model setup and an investment and dispatch optimization that uses the JUMP modeling language [@Dunning:2017]. An interface is provided between the optimization result and further analysis.
+
+# Infrastructure planning in the energy sector
+
+Energy systems convert different energy resources to meet desired demands like electric and thermal energy demands. Political, economical, and technological changes require changing the infrastructure of energy systems. Changing the infrastructure has to balance multiple political, environmental, and economical objectives. Capacity expansion planning can be an important tool during the planning process [@Gacitua:2018].
+
+Capacity expansion planning is used to compute cost-optimal energy system designs under given sets of constraints. The resulting cost-optimal energy system design can be used to inform policy decisions that incentivize the industry to invest in this design [@Johnston:2013]. Similarly, cost-optimal energy system designs can be used by companies for their investment strategies.
+
+Aspects of the energy system design that capacity expansion planning aims to answer are what the optimal technology mix is in regards of location, time, and installed generation, conversion, storage, and transmission capacities. The design optimization is done while using an integrated dispatch formulation to ensure that supply can equal demand at all nodes and time steps. The model determines the costs, emissions, power generation, energy storage, and power flows based on the installed capacities.
+
+Capacity expansion planning is formulated as mathematical optimization problem. Like any optimization problem, capacity expansion planning has certain degrees of freedom, consists of constraints, and consists of an objective function that is minimized: Typical degrees of freedom (sometimes called decision variables) are installed capacities, power generation, energy storage, and power flows. We divide constraints into internal and external constraints: The internal constraints ensure that the model is physically consistent in itself and following the rules of thermodynamics, e.g. energy balances ensure energy conservation over time. External constraints restrict the solution space to external conditions like costs, demands, available energy resources as well as other political, environmental, economical, or technological constraints. The objective function to determine cost-optimal investment can be total system cost, the net present value, or another business-oriented cost measure.
```

Comment:
```suggestion
Capacity expansion planning is formulated as a mathematical optimization problem. Like any optimization problem, capacity expansion planning has certain degrees of freedom, consists of constraints, and consists of an objective function that is minimized: Typical degrees of freedom (also called decision variables) are installed capacities, power generation, energy storage, and power flows. We divide constraints into internal and external constraints: The internal constraints ensure that the model is physically consistent in itself and following the rules of thermodynamics, e.g. energy balances ensure energy conservation over time. External constraints restrict the solution space to external conditions like costs, demands, available energy resources as well as other political, environmental, economical, or technological constraints. The objective function to determine cost-optimal investment can be total system cost, the net present value, or another business-oriented cost measure.
```

Reply:
Is the distinction between internal and external necessary? I find it slightly confusing, maybe take the distinction out and just list constraints.

Reply:
I agree, that it is not necessary.